README.md

```diff
@@ -22,7 +22,6 @@
 - [CSS Day](https://cssday.nl)
 - [DockerCon](https://dockercon.com/)
 - [EmberConf](https://emberconf.com/)
-- [FFConf](https://ffconf.org/)
```

Comment:
This is a European conference - it's also listed under Europe.

Reply:
good point